menu/menu.yaml

```diff
@@ -19,8 +19,10 @@
         Title: Publishing events
       - Url: tutorials/intro-to-nservicebus/5-retrying-errors
         Title: Retrying errors
-    - Url: tutorials/message-replay
-      Title: Message Replay Tutorial
```

Comment:
Is "Message Replay" completely gone?

Reply:
No, it's just that it's not really a "getting started" tutorial, same as the monitoring ones, so it can hide behind "More tutorials"